Mathlib/RingTheory/Polynomial/Chebyshev.lean

```diff
@@ -85,6 +86,25 @@ protected theorem induct (motive : ℤ → Prop)
   T.induct motive zero one add_two fun n hn hnm => by
     simpa only [Int.negSucc_eq, neg_add] using neg_add_one n hn hnm
 
+/-- Another induction principle used for proving facts about Chebyshev polynomials,
+    which is sometimes easier to use -/
+@[elab_as_elim]
+protected theorem induct' (motive : ℤ → Prop)
+    (zero : motive 0)
+    (one : motive 1)
+    (add_two : ∀ (n : ℕ), motive (↑n + 1) → motive ↑n → motive (↑n + 2))
```

Comment:
Is this the natural order? I would've expected
```suggestion
    (add_two : ∀ (n : ℕ), motive ↑n → motive (↑n + 1) → motive (↑n + 2))
```
but of course it doesn't matter too much.